about.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="./css/bootstrap.min.css" rel="stylesheet">
+    <script src="./js/bootstrap.bundle.min.js"></script>
+    <script src="https://kit.fontawesome.com/d2575abd28.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>Aseel Afghanistan</title>
+</head>
+<body class="about">
+    <!-- HEADER SECTION STARTS HERE -->
+    <header id="header" class="d-flex flex-column">
+        <nav class="navbar navbar-expand-sm" id="black-navbar">
+            <div class="container">
+                <div class="collapse navbar-collapse first-navbar-list" id="navbarNav">
+                    <div class="navbar-nav">
+                        <a class="navbar-brand" href="#"><li class="fa fa-facebook"></li></a>
+                        <a class="navbar-brand" href="#"><li class="fa fa-twitter"></li></a>
+                        <a class="navbar-brand" href="#">English</a>
+                        <a class="navbar-brand" href="#">My Page</a>
+                        <a class="navbar-brand" href="#">Logout</a>
+                    </div>
+                </div>
+            </div>
+        </nav>
+        <nav class="navbar navbar-expand-lg bg-body-tertiary">
+            <div class="container-fluid">
+                <a class="navbar-brand" href="#"><img src="./images/aseel-logo.webp" alt="logo" id="logo"></a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+                <span id="closeNav" class="fa fa-bars"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarNav">
+                    <ul class="navbar-nav">
+                    <li class="nav-item">
+                        <a class="active" aria-current="page" href="index.html">About</a>
```

Comment:
- [ ] It will be better to change the text from `About` to `Home` to not confuse the recruiter in the future when check your portfolio